.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,71 @@
+name: "Request a Feature"
+description: Request a change for SFML
+labels: ["feature"]
+body:
+    - type: markdown
+      attributes:
+          value: |
+              Thank you for bringing up this idea for discussion! 🙂
+
+    - type: checkboxes
+      attributes:
+          label: Prerequisite Checklist
+          description: "Before you submit the issue, we'd like you to consider the follow points."
+          options:
+              - label: "I searched for [existing issues](https://github.com/search?q=repo%3ASFML%2FSFML&type=issues) to prevent duplicates"
+                required: true
+              - label: "I searched for [existing discussions on the forum](https://www.google.com/search?q=site%3Ahttps%3A%2F%2Fen.sfml-dev.org) to prevent duplicates"
+                required: true
+              - label: "I understand [SFML's scope](https://www.sfml-dev.org/contribute.php#general-considerations) and believe the feature fits it"
+                required: true
+              - label: "I am aware and accept that my feature request can also be declined"
+                required: true
```

Comment:
I guess since the number of "entitled" requests is kind of low, it might make sense to not have it here. It was more to get ahead of people who think, because they've opened a request, that it will now be implemented.

Reply:
I removed it 🙂